tests/ct_connection.c

```diff
@@ -134,9 +204,10 @@ void test_async_password_wrong() {
     assert(acc->cc->err == REDIS_ERR_OTHER);
     assert(strncmp(acc->cc->errstr, "WRONGPASS", 6) == 0);
 
-    // Test connection
-    status = redisClusterAsyncCommand(acc, commandCallback, (char *)"THE_ID",
-                                      "SET key1 Hello");
+    // No connection
+    ExpectedResult r;
+    status =
+        redisClusterAsyncCommand(acc, commandCallback, &r, "SET key1 Hello");
```

Comment:
Perhaps a matter of personal taste, but I suggest we keep the old layout with the linebreak before `"SET key1 Hello"`. Or is this clang-format's preferred style?

Reply:
Yes, clang-format is changing it to the worse.. maybe we should think about if we can live without it..
I changed a bit so it looks ok again..